### ISSUE TITLE
Automate access to high-level Moped data points for TPW Stat

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#15506]

**Comment by Charlie-Henry** [comment]
@johnclary do you think any of these performance measures could be automatically reported on from the Moped RR?

**Comment by johnclary** [comment]
yes. are y'all getting this reporting needs from Moped stakeholders? with ATSD in particular, they already have a number of metrics that they either currently report on or are in discussion.

**Comment by Charlie-Henry** [comment]
Yeah @michael-santulli is getting their reporting manually. We're looking at making at automating as many of these TPWstat performance measures.

**Comment by michael-santulli** [comment]
@Charlie-Henry @johnclary  yeah the current process is requesting the performance data from Laura, adding to a spreadsheet in sharepoint that is connected to PowerBI.

**Comment by johnclary** [comment]
Great. we will need to get with Nathan W., who almost definitely hands the data to Laura. he and I have a running list of Moped reporting todos, which is quite sprawling and needs to be cleaned up + prioritized since the last batch of updates we shipped.
